test/pytest.ini

```diff
@@ -0,0 +1,20 @@
+[pytest]
+asyncio_mode = auto
+
+log_cli = true
+log_format = %(asctime)s.%(msecs)03d %(levelname)s>  %(message)s
+log_date_format = %H:%M:%S
+
+markers =
+    slow: tests that take more than 30 seconds to run
+    replication_factor: replication factor for RandomTables
```

Comment:
These markers don't look to well thought out, but I guess you just copied them from test/topology, so what the heck...

Reply:
Correct this is markers that are used in topology and auth_cluster tests. At least there were in the pytest.ini